app/test/e2e/floating-ip-create.e2e.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, you can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * Copyright Oxide Computer Company
+ */
+
+import { clickRowAction, expect, expectNotVisible, expectVisible, test } from './utils'
+
+const floatingIpsPage = '/projects/mock-project/floating-ips'
+
+test('can create a Floating IP', async ({ page }) => {
+  await page.goto(floatingIpsPage)
+  await page.locator('text="New Floating IP"').click()
```

Comment:
I try to use `getByRole` for things like this as it lets us exercise the [accessible name](https://developer.mozilla.org/en-US/docs/Glossary/Accessible_name) and ARIA role of the element.

```suggestion
  await page.getByRole('link', { name: 'New Floating IP' }).click()
```